src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        return Colors.values()[new Random().nextInt(Colors.values().length)].toString();
```

Comment:
You don't need to create new `Random` instance every time you call `getRandomColor`, introduce it as field variable `random`.
And then provide `random.nextInt(Colors.values().length)` as local variable to make your code more readable.